nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/OperationListCell.java

```diff
@@ -160,26 +160,17 @@ public void handle(DragEvent event) {
 
                 /* the drag and drop gesture ended */
  /* if the data was successfully moved, clear it */
-                if (event == null || event.getTransferMode() == null) {
-                    ;
-                } else if (target.getIndex() < 0 || target == null) {
-                    ;
-                } else if (event.getTransferMode() == TransferMode.COPY) {
-                    int sourceIndex = Math.max(0, source.getIndex());
-                    int targetIndex = Math.min(target.getIndex(), listItems.size() - 1);
-                    //System.out.println("drag done source " + sourceIndex + " to " + targetIndex);
-                    String swap;
-                    //ObservableList<String> listItems = MainApp.mainController.listItems;
-
-                    swap = listItems.remove(sourceIndex);
-                    listItems.add(targetIndex, swap);
-                    scriptView.getSelectionModel().select(targetIndex);
-// fixme
-//                    processorController.chartProcessor.execScriptList();
-//                    processorController.propertyManager.chartProcessor.getChart().layoutPlotChildren();
-//                    processorController.propertyManager.popOver.hide();
```

Comment:
These do say fix me, but they have been commented out since 2017...I can add them back in if needed.